games-simulation/openrct2/openrct2-0.1.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="An open source re-implementation of RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.website/"
+MY_REPO="https://github.com/OpenRCT2/OpenRCT2"
+if [ "${PV}" == "9999" ]; then
+	EGIT_REPO_URI="${MY_REPO}.git"
+	EGIT_BRANCH="develop"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="${MY_REPO}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/OpenRCT2-${PV}"
+fi
+
+TSV=0.1.0
+SRC_URI+=" https://github.com/OpenRCT2/title-sequences/releases/download/v$TSV/title-sequence-v$TSV.zip -> ${PN}-title-sequence-v$TSV.zip "
```

Comment:
Use `${...}` consistently. I don't really understand why `$TSV` would be the only var not having it ;-P.